### ISSUE TITLE
CustomRoute and TransitionBuilders fails to build

### ISSUE THREAD

**Issue by myklll** [Stacked-Org/stacked#349]
Running `flutter pub run build_runner build --delete-conflicting-outputs` fails when you use  a CustomRoute with one of the TransitionBuilders with the error 
```
[SEVERE] stacked_generator:stackedRouterGenerator on lib/app/app.dart:

type 'Null' is not a subtype of type 'String'
```

**Comment by myklll** [comment]
@FilledStacks Do you have some time to look at this? Thank you.

**Comment by myklll** [comment]
@hurbes Please can you take a look at this? Thank you.

**Comment by FilledStacks** [comment]
@mychell I'll need more than that to get this fixed up. 

1. What version of stacked and stacked_generator are you on?
2. Post your app.dart file code here

**Comment by ajnaf** [comment]
I think this still an issue for versions of the stacked generator the issue is with the custom route when adding a transitionBuilders to it ,
This is a duplicate of my issue.#310

**Comment by FilledStacks** [comment]
Fixed in #357 . Will push update in a few minutes
